src/components/eventSingleComponent/image.ts

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
This doesn't make sense to me. Its defining a class named kdImage but it seems to be virtually the same content as the eventSingleComponent. If it is needed, it should be named something different than kdImage